### PR TITLE
2nd Change to document 1

### PR DESCRIPTION
**Pull request by E1000y**
### Refence a related issue

This pull request solves #2

### Description of the changes

- Project now prints sentiment as a number
- A description of what the sentiment number is has been added

### Mentions

Person responsible for reviewing: @LaloCo

### CODE REVIEW

**Review by E1000y — COMMENTED** [review]
all wrong, let's go.

**Review by E1000y — COMMENTED** [review]
let's approve this

File: TextAnalytics.ipynb [hunk 1]
```diff
@@ -1 +1 @@
...Qaw56EQvYIAiCIAiCIBgB4Q8cBMMnFrBBEARBEARBEATBVDCn6Q4EQRAEQRAEQRAEQQ6xgA2CIAiCIAiCIAimgljABkEQBMGIkHSUpPsk3S3pTknbSHqvpNWa7lsQBEEQTCNRAxsEQRAEI0DSS4DPAjuY2RJJ6wBPAm4EtjKzRxrtYBAEQRBMIbEDGwRBEASj4RnAI2a2BCAtWPcAngnMlzQfQNJrJN0k6XZJ50mam57/maSPp9/dJmmepMsl/VjSwek1z5B0XdrdvVfSy5r5qEEQBEEwHmIHNgiCIAhGQFqILgBWA64CzjGzayX9jLQDm3ZlvwXsbGa/lXQEsLKZHZted7yZnSLpc8BOwHbAKsB9ZvY0Se8HVjGz4yStAKxmZovH/mGDIAiCYEys2HQHgiAIgmA2YmaPSdoSeBmwI3COpCO7XrYtsBlwgyTwFOOb2n5/Ufp5DzA3LU4XS/qDpLWAW4F/kbQS8G0zu3N0nygIgiAImicWsEEQBEEwIszsT8A1wDWS7gEO6HqJgCvNbO8Z3mJJ+vnntv8Xj1c0s+skvRx4A3CmpE+Z2VeH9gGCIAiCYMKIGtggCIIgGAGSNpH0vLantgB+DiwGVk/PLQS2k/Tc1GY1Sc8vcYz1gYfM7MvAV4B5Q+l8EARBEEwosQMbBEEQBKNhLnBySvV9AvhP4G+BvYFLJT1oZjtKehvwdUkrp3YfBv4j8xg7AIdLehx4DNh/iP0PgiAIgokjRJyCIAiCIAiCIAiCqSBSiIMgCIIgCIIgCIKpIBawQRAEQRAEQRAEwVQQC9ggCIIgCIIgCIJgKogFbBAEQRAEQRAEQTAVxAI2CIIgCIIgCIIgmApiARsEQRAEQRAEQRBMBbGADYIgCIIgCIIgCKaCgTCm0AAAAAZJREFU/wNAv1CB5oDHLgAAAABJRU5ErkJggg==\n","text/plain":["<Figure size 1152x648 with 1 Axes>"]},"metadata":{"tags":[],"needs_background":"light"}}]},{"cell_type":"code","metadata":{"trusted":false,"id":"3Z0P4Lg6ih31","colab_type":"code","colab":{}},"source":[""],"execution_count":0,"outputs":[]}]}
```
Comment (E1000y) [inline TextAnalytics.ipynb:1]: comment for fun

**Review by E1000y — COMMENTED** [review]
hello hello we're good to go

File: TextAnalytics.ipynb (hunk 1 above)
Comment (EmiliePaniagua) [reply TextAnalytics.ipynb:1]: reply for kicks

**Review by EmiliePaniagua — APPROVED** [review]

**Pull request merged**